Closets/Crates are now puncheable

emmanuelbassil opened the pull request:
🆑 emmanuelbassil
tweak: As part of the damage standardization, can now damage closets/crates with melee attacks.
tweak: Damaging a closet also relays some damage to whatever's inside.
tweak: Secure closets/crates are tougher than regular closets/crates.
/🆑 

I also changed breakable (which is unused at the moment) to become 'fragile'. This var instantly kills a structure if hit by hulk or a turf breaker. 
Technically you *could* punch a closet open, but given for a regular non-hulk; non-clawed human 60% of attacks succeed on regular closets and 20% on secure closets, you're more likely to break your hand angrily punching metal. Also applied this to large crates, which you can now punch or stun baton open.

emmanuelbassil commented:
This is ready.
Also copied some code from gift wrapping; damaging closets also translates some of the damage inside.

SierraKomodo commented on code/game/objects/objs.dm:
```diff
@@ -242,6 +242,8 @@
 
 
 /obj/attack_hand(mob/living/user)
+	if ((. = ..()))
```
```suggestion
	if (. = ..())
```

SierraKomodo replied:
```suggestion
	if (. = ..())
```

emmanuelbassil committed:
Allows to punch closets

Fixes

Auto stash before rebase of "refs/heads/Closetfix"

Fxes

Auto stash before rebase of "refs/heads/CtrlAltClick"

Auto stash before rebase of "refs/heads/Closetfix"

Auto stash before rebase of "refs/heads/Crowfix"

Auto stash before rebase of "refs/heads/Allergies"

emmanuelbassil commented:
Fixed

SierraKomodo reviewed (approved):
After some local testing this seems to work and doesn't appear to break anything else. Test merge probably isn't necessary here.
